frontend/src/utils/serializeFilterParams.js

```diff
@@ -7,6 +7,10 @@ import {FILTER_TYPE} from "../constants"
 export default function serializeFilterParams(filters, descriptions) {
   const params = {}
 
+  function hasWhiteSpace(s) {
+    return /\s/g.test(s);
```

Comment:
This may not be the most targeted this can be... it would be better to check specifically for space instead of generic white space (including tab and newline). If your filter just split by " " you should test for that ... or change the filter to accept the other white space characters as separators...